paperoni/webapp/validation.py

```diff
@@ -19,6 +20,9 @@ async def app(page, box):
     area = H.div["area"]().autoid()
     papers = {}
 
+    user = page.session.get("user", {}).get("email", None)
+    update_logger_handler(db_logger, user)
```

Comment:
This is not going to work. This is an async function and its lifetime is the whole time the user is using the validation interface. If two different users are using the interface, this function will be called twice and both instances will run in parallel and yield to each other through the event loop. The second user will therefore override `handler.user` and all changes from both users will show up as being from the second.

If you move the statement to be executed immediately prior to any database change, it should work, but I'd rather revert to the first version of the PR.

Reply:
will do the change

Reply:
should be good now

Reply:
Thank you!